lib/jxl/decode.cc

```diff
@@ -2183,8 +2183,6 @@ JxlDecoderStatus JxlDecoderSetPreferredColorProfile(
   return JXL_DEC_SUCCESS;
 }
 
-#if JXL_IS_DEBUG_BUILD || defined(JXL_ENABLE_FUZZERS)
 void SetDecoderMemoryLimitBase_(size_t memory_limit_base) {
```

Comment:
So, this function will become part of almost public API? @lvandeve WDYT?

Reply:
No, this is not exported and won't be on the shared library. The fuzzer_runner is using the static library which has access to all the internal symbols.

A functionality to limit the memory usage could be added to the API, or gracefully handling of OOM conditions could be added to the library functions. I'm not sure this particular function is the best approach to handle the situation and it was a workaround that needs cleanup.